source/extensions/transport_sockets/tls/context_config_impl.cc

```diff
@@ -19,6 +21,30 @@ namespace Tls {
 
 namespace {
 
+absl::once_flag fipsOnce;
+bool fipsEnabled;
+
+void initFipsEnabled() {
+  std::ifstream file("/proc/sys/crypto/fips_enabled");
+  if (file.fail()) {
+    std::cout << "FIPS disabled\n";
```

Comment:
Could you replace cout's with `ENVOY_LOG(debug, "message here")`? You'll need to include `"common/common/logger.h"` and add `"//source/common/common:logger_lib"` dependency to the `context_lib` target in `source/extensions/transport_sockets/tls/BUILD`.

Reply:
No, I had that initially and it's at warning level so we don't see it.

Reply:
I don't want to set it at warning, or higher, to guarantee it is displayed every time

Reply:
Would info level work here?

Reply:
No, that's what I had but the default settings are for warning and above.  @jwendell came up with a better solution and I'll try to implement that in a separate PR.